src/App.js

```diff
@@ -1,11 +1,32 @@
-import React from 'react';
-
-function App() {
-  return (
-    <h1>
-      Hello Wors
-    </h1>
-  );
+import React, { Component } from 'react';
+import Header from './Components/Header/Header';
+import Form from './Components/Form/Form';
+import WeatherDetails from './Components/WeatherDetails/WeatherDetails';
+
+class App extends Component {
+  state = {
+    weatherData: '',
+  };
+
+  handleState = data => {
+    this.setState({
+      weatherData: data,
+    });
+  };
+
+  render() {
+    const { weatherData } = this.state;
+
+    return (
+      <>
+        <Header />
+        <div className="container">
+          <Form handleState={this.handleState} />
```

Comment:
dont name props as handle

handle is a method of a component that handles somethings.
Convention to name props is to use prefix onSomething

onChange, onClick, onError etc.

please change this prop name

Reply:
I have change it to setWheaterData={this.setWheaterData} cuz I'm pasting setWeatherData in to form component and I use it my props in there.
this.props.setWheaterData(response)

so in Form.js I know whot is going on.